src/aap_eda/services/activation/engine/kubernetes.py

```diff
@@ -158,8 +158,9 @@ def get_status(self, container_id: str) -> ActivationStatus:
                 "set status to error.",
             )
             status = ActivationStatus.ERROR
+            message = f"Pod {container_status} is not running or terminated"
```

Comment:
Same as my previous comments. I think all these messages should be placed in messages.py and shared by both engines for consistency.

Reply:
Also I think it should be "is not running or **is** terminated"